flopy/modpath/mp7.py

```diff
@@ -137,20 +137,14 @@ def __init__(
                 )
             else:
                 if dis.package_name.lower() == "dis":
-                    nlay, nrow, ncol = (
-                        dis.nlay.array,
-                        dis.nrow.array,
-                        dis.ncol.array,
-                    )
+                    nlay, nrow, ncol = (dis.nlay.array, dis.nrow.array, dis.ncol.array)
                     shape = (nlay, nrow, ncol)
                 elif dis.package_name.lower() == "disv":
                     nlay, ncpl = dis.nlay.array, dis.ncpl.array
                     shape = (nlay, ncpl)
                 elif dis.package_name.lower() == "disu":
                     nodes = dis.nodes.array
-                    shape = tuple(
-                        nodes,
-                    )
+                    shape = tuple(nodes)
```

Comment:
This should probably be corrected to `shape = (nodes,)` (it was probably originally `shape = tuple(nodes,)` which is not correct)